### ISSUE TITLE
Can use character input ?

### ISSUE THREAD

**Issue by bino282** [jaywalnut310/vits#18]
i want train this model with japanese, i can use character as input
Thank

**Comment by nikich340** [comment]
Yes you can, but then it may be harder for model to learn prononcuation rules if the same symbol match different signals (while phoneme usually match the same signals).

**Comment by himekifee** [comment]
I'm quite interested in this topic though I don't quite have that much knowledge in the field. Could you briefly tell me what is limited in the scope of enabling character pronunciations, please? Does network architecture need a change for the task? I tried to ask GPT, and GPT4 seems pretty good at producing the symbolic rule but still needs tunning. It seems a possible way to coordinate with llm to produce a better result for this task.
